setup.py

```diff
@@ -51,6 +51,7 @@
 
 extras_require = {
     # Readers:
+    'avhrr_l1b_gaclac': ['pygac'],
```

Comment:
Should we give version 1.2.0 as minimum ? (the version we will be releasing with your changes)

Reply:
Done!